overworld/TileMap.gd

```diff
@@ -187,8 +187,13 @@ func place_exit(startingSpot):
 	var offsetVariance = 2
 	var cellIndex
 	var ladderPlaced = false
+	var loops = 0
 	
-	while !ladderPlaced: 
+	while !ladderPlaced:
+		loops += 1
+		if loops >= 12:
```

Comment:
Arbitrary 12!?!?! MAGIC NUMBERS EVERYWHERE!!!!!

Reply:
its literally an arbitrary number. Tried a couple numbers till i found one that felt pleasant in loading time